.env

```
@@ -0,0 +1,7 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
```

どうせ後で書き換えるとはいえMySQL使うならついでに書き換えちゃいましょう
ファイルの末尾に改行あるといいです。editorconfig入れとくとその辺勝手にやってくれてお得
```suggestion
DATABASE_URL = "mysql://johndoe:randompassword@localhost:3306/mydb"

```

---

末尾改行の設定してる箇所は以下です

https://github.com/ih14c-13-14/artist-backend/blob/5cded557b3b0d2cd81400e09596e33a3fae3d5aa/.editorconfig#L6